src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -105,8 +105,8 @@ public void initBinder(WebDataBinder webdataBinder) {
     @PostMapping(value = "/studies/cases/{caseUuid}")
     @Operation(summary = "create a study from an existing case")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = "The id of the network imported"),
-        @ApiResponse(responseCode = "409", description = "The study already exist or the case doesn't exists")})
+            @ApiResponse(responseCode = "200", description = "The id of the network imported"),
```

Comment:
keep previous indentations (this code has not been changed ...)
same remarks in following lines